pkg/metrics/metrics.go

```diff
@@ -19,12 +19,15 @@ const (
 	assignedIdentityDeletionDurationName   = "assigned_identity_deletion_duration_seconds"
 	assignedIdentityDeletionCountName      = "assigned_identity_deletion_count"
 	nmiOperationsDurationName              = "nmi_operations_duration_seconds"
+	nmiTokenOperationCountName             = "nmi_token_operation_count"
```

Comment:
Would you not need a success count?

Reply:
Cant, we calculate the success count by total - failure?